bioframe/ops.py

```diff
@@ -1009,7 +1009,8 @@ def _closest_intidxs(
         ).T
 
         closest_intidxs.append(closest_idxs_group)
-
+    
+    if len(closest_intidxs)==0: return []
```

Comment:
Ah, thank you! Probably, better return np.ndarray(shape=(0,2), dtype=np.int) instead, to keep the type+shape the same